docs/guide/user_management/user_management.md

```diff
@@ -17,6 +17,41 @@ You can [customize it according to your needs](#customize-login-form).
 The validity of the password recovery token can be set using the `ezplatform.system.<siteaccess>.security.token_interval_spec` parameter.
 By default it is set to `PT1H` (one hour).
 
+### Add forget password option in site front
+
+You can add a forgot password option to the site front to allow users of a specific site request a password change.
+
+Edit `congfig/packages/ezplatform.yaml`. The templates are defined with the following configuration:
+
+```yaml
+ezplatform:
+    system:
+        <siteaccess>:
+            user_forgot_password:
+                templates:
+                    form: <path_to_template>
```

Comment:
Maybe one sentence explaining what the two templates are responsible for? Obvious, I know, but...